lib/legacy.coffee

```diff
@@ -1,3 +1,5 @@
+fs = require('fs')
```

Comment:
@iam4x there's a variable named path right below this line and I've seen a lot of function use that name.